sdk/cosmos/azure-cosmos-spark_3-1_2-12/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 
 #### Bugs Fixed
 * Fixed an issue with creating new Throughput control client item when `enableThroughputControlGroup` is being called multiple times with the same throughput control group. - See [PR 28905](https://github.com/Azure/azure-sdk-for-java/pull/28905)
+* Fixed a possible dead-lock on static ctor for CosmosException when the runtime is using custom class loaders. - See [PR 28912](https://github.com/Azure/azure-sdk-for-java/pull/28912)
```

Comment:
We should also mention this in the azure-cosmos changelog.

Reply:
Fixed (next iteration)